Bermuda/Bermuda/AggressiveBehaviour.cpp

```diff
@@ -0,0 +1,45 @@
+#include "AggressiveBehaviour.h"
+#include "PlayState.h"
+
+AggressiveBehaviour::AggressiveBehaviour(StateMachine<Entity>* m_pStateMachine) :
+	BaseBehaviour(m_pStateMachine)
+{
+
+}
+
+void AggressiveBehaviour::executeAction()
+{
+	
+	dynamic_cast<AttackingNPC*>( this->npc )->attack();
```

Comment:
Is het niet slimmer om een AttackingNPC variable bij te houden. (die je in de constructor cast vanuit je NPC). 
Op deze manier doet hij elke frame dat hij aanvalt, deze cast uitvoeren.